src/style-manager.js

```diff
@@ -144,50 +144,56 @@ module.exports = class StyleManager {
       }
     }
 
-    if (params.skipDeprecatedSelectorsTransformation) {
-      styleElement.textContent = source;
-    } else {
-      const transformed = this.upgradeDeprecatedSelectorsForStyleSheet(
-        source,
-        params.context
+    let textContent;
+    let deprecationMessages = [];
+
+    textContent = source;
```

Comment:
```suggestion
    let textContent = source
    let deprecationMessages = [];
```

Tiny cleanup, if you like the suggestion go for it, otherwise can leave it as-is!

Reply:
Yeah good call here, thanks!